lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -488,8 +493,8 @@ if (require.main === module) {
   console.log('Collected from Stack Packs!');
 
   if ((collisions) > 0) {
-    assert.equal(collisions, 15, 'The number of duplicate strings have changed, update this assertion if that is expected, or reword strings.');
     console.log(`MEANING COLLISION: ${collisions} string(s) have the same content.`);
+    assert.equal(collisions, 17, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings}`);
```

Comment:
URL and....?

Reply:
...and URL again.  I count each string in the collision, so, URL is a two way collision, hence +2.  This is because some collisions are actually 3 way collisions, and that was lost if only counting the raw number of collisions instead of the number of strings colliding.  It is weird.

Reply:
ah, gotcha!